src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {
+            "червоний", "зелений", "синій", "жовтий", "чорний"};
```

Comment:
Use only English in messages/code. The color names should be in English to make the code universal and consistent.

Reply:
Fix this and use enum for colors